test/e2e/specs-playwright/wp-log-in-out-spec.js

```diff
@@ -17,21 +16,15 @@ const mochaTimeOut = config.get( 'mochaTimeoutMS' );
 describe( `Auth Screen @canary @parallel`, function () {
 	this.timeout( mochaTimeOut );
 
-	// BrowserContext is equivalent to the `driver` used in Selenium.
-	let browserContext;
 	// Page represents a tab in a browser.
 	// Test steps interact with the page to execute its instructions.
 	let page;
 
-	before( 'Start browser', async function () {
-		browserContext = await BrowserManager.newBrowserContext();
-	} );
-
-	beforeEach( 'Open new test tab', async function () {
-		page = await browserContext.newPage();
-		// Set the page using mocha's metadata. Upon test failure,
-		// mocha hooks can access the page to perform actions.
-		this.currentTest.page = page;
+	// Open a new page (tab) within the BrowserContext.
+	// This is akin to the following call in the Selenium test:
+	// driver = await driverManager.startBrowser();
+	before( 'Open new test tab', async function () {
+		page = await this.browserContext.newPage();
```

Comment:
Out of curiosity, will this have to be added for all tests? If so, does it make sense to abstract it more?

Reply:
I would like to abstract it some more, definitely. In my previous attempts I ended up running into issues with mocha contexts, but I'll take another crack at it!

Reply:
Since my last message there was the big slack thread about Mocha + Playwright ([here](p1620084874022800-slack-C01QG4Y91RR)).

I think I was able to come to a reasonable level of abstraction for the time being, where hooks:
- launch the browser
- create a new browser context
- create a new page, which is where the tests execute in

As Sergio noted in the slack thread, in the future it would be a good idea to override the behavior of describe blocks with our own `e2e_describe` that encapsulates the actions listed above. However, that is not within the scope of this PR.

Reply:
That sounds good to me!